addons/website/controllers/backend.py

```diff
@@ -36,3 +38,13 @@ def fetch_dashboard_data(self, website_id, date_from, date_to):
     @http.route('/website/iframefallback', type="http", auth='user', website=True)
     def get_iframe_fallback(self):
         return request.render('website.iframefallback')
+
+    @http.route('/website/check_new_content_access_rights', type="json", auth='user')
+    def check_create_access_rights(self, models):
+        if not request.env.user.has_group('website.group_website_restricted_editor'):
+            raise werkzeug.exceptions.Forbidden()
+
+        access_rights = {}
+        for model in models:
+            access_rights[model] = request.env[model].check_access_rights('create', raise_exception=False)
+        return access_rights
```

Comment:
```suggestion
return {
    model: request.env[model].check_access_rights('create', raise_exception=False)
    for model in models
}
```
(really just a suggestion, I don't mind either way but I guess it's more pythony to do it that way :))

Reply:
I used your pythony way. 😄 I don't always find it cleaner, but in this case, I feel like it is and makes it clear what we actually return from this route.